gnovm/pkg/gnolang/stacktrace.go

```diff
@@ -0,0 +1,78 @@
+package gnolang
+
+import (
+	"fmt"
+	"strings"
+)
+
+func (m *Machine) buildStackTrace() string {
+	var builder strings.Builder
+
+	index := len(m.Stmts)
+	for i := len(m.Frames) - 1; i >= 0; i-- {
+		if m.Frames[i].Func != nil {
```

Comment:
Yes, but I cannot run a test for GoFunc because we have not addressed the golang runtime panic. You can find more information at: https://github.com/gnolang/gno/issues/2146